libs/designsystem/menu/src/menu.component.ts

```diff
@@ -34,9 +38,12 @@ import { EventListenerDisposeFn } from '@kirbydesign/designsystem/types';
   styleUrls: ['./menu.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MenuComponent implements AfterViewInit, OnDestroy {
+export class MenuComponent implements AfterViewInit, AfterContentInit, OnDestroy {
+  readonly MENU_ID: string = 'MENU_ID';
+  triggerButtonId: string = 'DEFAULT_BUTTON';
```

Comment:
We should ensure the id is unique across the page when more then one menu is present:
```suggestion
  triggerButtonId: string = UniqueIdGenerator.scopedTo('kirby-menu-trigger-button').next();
```